pkg/csi/service/vanilla/controller.go

```diff
@@ -565,6 +565,10 @@ func (c *controller) createBlockVolume(ctx context.Context, req *csi.CreateVolum
 			return nil, csifault.CSIInternalFault, logger.LogNewErrorCodef(log, codes.Internal,
 				"failed to get shared datastores in kubernetes cluster. Error: %+v", err)
 		}
+		if len(sharedDatastores) == 0 {
+			return nil, csifault.CSIInternalFault, logger.LogNewErrorCode(log, codes.Internal,
+				"shared datastore not found.")
```

Comment:
can we change this message to "No datastore found for volume provisioning".

Reply:
fixed.